services/core-api/app/api/mines/incidents/resources/mine_incidents.py

```diff
@@ -125,6 +127,7 @@ def post(self, mine_guid):
         incident = MineIncident.create(
             mine,
             data['incident_timestamp'],
+            data['incident_timezone'],
```

Comment:
Will the timestamp be stored with the offset included, or will it be converted into UTC? (Ideally the latter 😅 )

Reply:
On the frontend I do a .toISOString() which converts it to UTC before sending it over (in the formatPayload function of MineIncident.js for Core).
Fun(?) thing I learned about the database:
If you go into the table for this field and manually change the offset, the value will adjust itself to have an offset of 0 so it always stays as UTC (so if I add an offset of 1hr, it will adjust the time by 1hr, and then save it with an offset of 0).

Reply:
Ah missed that one.

Oo that's a fun fact! I had no idea that was a thing, pretty neat 📅